types/react-confirm/index.d.ts

```diff
@@ -1,15 +1,14 @@
 // Type definitions for react-confirm 0.1
 // Project: https://github.com/haradakunihiko/react-confirm
 // Definitions by: santiagodoldan <https://github.com/santiagodoldan>
+//                 Mark Nelissen <https://github.com/marknelissen>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import * as React from "react";
 
-export class PlainConfirmModal extends React.Component<any, any> { }
```

Comment:
Is removing the exported class intentional?

Reply:
Yes. It only served as typing for the arguments of one of the two actual exports. The JS library only exports the methods `confirmable` and `createConfirmation`. Since the above type is no longer used in their definition, I removed it.

I think nobody used that definition in their code, and if they did, it's a bad idea, since it doesn't expose the generic typing of the `Component` class it extends.